### PR TITLE
IEEE 202: Create Header on Orders Page

### PR DESCRIPTION
**Pull request by KarandeepLubana**
## Overview

- Resolves <[IEEE-202](https://ieeeuoft.atlassian.net/jira/software/projects/IEEE/boards/1?selectedIssue=IEEE-202)>
- Created the Orders Page Title in Orders component
- Created Orders Search Bar in a separate component under `/src/components/orders/OrdersSearch`
- Created filter button in a separate component under `/src/components/orders/OrdersFilter`
- Created total count label and reload button all under a separate component in `/src/components/orders/OrdersCount`
- All the created components were put together in Orders component

## Unit Tests Created

- Tested the OrdersSearch.tsx, OrdersFilter.tsx, OrdersCount.tsx and Orders.tsx for containing all necessary elements

## Steps to QA

- Navigate to http://localhost:3000/orders

### CODE REVIEW

File: hackathon_site/dashboard/frontend/src/pages/Orders/Orders.tsx
Comment (Leo6Leo) [inline hackathon_site/dashboard/frontend/src/pages/Orders/Orders.tsx:1]: I think there are some problems while I am trying test your ticket Karandeep. And it might be caused by the conversion from JS to Typescript. 

CleanShot 2022-06-24 at 00 25 27@2x

Comment (YuYing-Liang) [reply hackathon_site/dashboard/frontend/src/pages/Orders/Orders.tsx:1]: Leo, I think you may have changes on your side because it worked fine for me. Maybe do a git stash before pulling?

Comment (Leo6Leo) [reply hackathon_site/dashboard/frontend/src/pages/Orders/Orders.tsx:1]: It magically works when I woke up in the morning. Thanks Yuying!

File: hackathon_site/dashboard/frontend/src/components/orders/OrdersFilter/OrdersFilter.tsx
Comment (Leo6Leo) [inline hackathon_site/dashboard/frontend/src/components/orders/OrdersFilter/OrdersFilter.tsx:3]: If you could switch line 3 and line 4, to put all MUI stuff together, that would be great.

File: hackathon_site/dashboard/frontend/src/components/orders/OrdersSearch/OrdersSearch.tsx
Comment (Leo6Leo) [inline hackathon_site/dashboard/frontend/src/components/orders/OrdersSearch/OrdersSearch.tsx]: same here. Move formikValue to the bottom

**Review by Leo6Leo — DISMISSED** [review]
Great Job Karandeep!! Your code looks fine to me. And I have also tested on my end, it works without any error. Wait for a final review from Yuying.

**Review by YuYing-Liang — DISMISSED** [review]

**Review by YuYing-Liang — APPROVED** [review]

**Pull request merged**
